src/tv2-common/helpers/graphics/pilot/index.ts

```diff
@@ -47,19 +47,26 @@ export interface PilotGeneratorSettings {
 	viz: VizPilotGeneratorSettings
 }
 
+export interface PilotGraphicProps {
+	config: TV2BlueprintConfig
+	context: IShowStyleUserContext
+	engine: GraphicEngine
+	partId: string
+	parsedCue: CueDefinitionGraphic<GraphicPilot>
+	settings: PilotGeneratorSettings
+	adlib: boolean
+	segmentExternalId: string
+	adlibRank?: number
```

Comment:
Is there a dependency between `adlib` and `adlibRank`?
If there is, it could be nice to model it in the types. E.g.:
```typescript
   adlib?: { rank: number }
```